<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ElectricalManager</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>2</y>
     <width>901</width>
     <height>531</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>用电类型</string>
    </attribute>
    <widget class="QTableView" name="tableViewEtype">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>600</width>
       <height>400</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>41</number>
     </attribute>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>100</y>
       <width>160</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_query_Etype">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_delete_Etype">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_insert_Etype">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_revert_Etype">
        <property name="text">
         <string>撤销</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_updata_Etype">
        <property name="text">
         <string>提交修改</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>业务员</string>
    </attribute>
    <widget class="QTableView" name="tableViewClerks">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>600</width>
       <height>400</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>100</y>
       <width>160</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_query_Clerks">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_delete_Clerks">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_insert_Clerks">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_revert_Clerks">
        <property name="text">
         <string>撤销</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_updata_Clerks">
        <property name="text">
         <string>提交修改</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>客户信息</string>
    </attribute>
    <widget class="QTableView" name="tableViewClients">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>600</width>
       <height>400</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>41</number>
     </attribute>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>100</y>
       <width>160</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_query_Clients">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_delete_Clients">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_insert_Clients">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_revert_Clients">
        <property name="text">
         <string>撤销</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_updata_Clients">
        <property name="text">
         <string>提交修改</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>客户用电信息</string>
    </attribute>
    <widget class="QTableView" name="tableViewEusedT">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>600</width>
       <height>400</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>41</number>
     </attribute>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>90</y>
       <width>160</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QPushButton" name="pushButton_query_EusedT">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_delete_EusedT">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_insert_EusedT">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_revert_EusedT">
        <property name="text">
         <string>撤销</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_updata_EusedT">
        <property name="text">
         <string>提交修改</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>费用管理</string>
    </attribute>
    <widget class="QTableView" name="tableViewCostManage">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>600</width>
       <height>400</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>41</number>
     </attribute>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>100</y>
       <width>160</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QPushButton" name="pushButton_query_CostManage">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>收费登记</string>
    </attribute>
    <widget class="QTableView" name="tableViewCharge">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>600</width>
       <height>400</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>41</number>
     </attribute>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_9">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>60</y>
       <width>160</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QPushButton" name="pushButton_query_Charge">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_delete_Charge">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_insert_Charge">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>调用存储过程</string>
    </attribute>
    <widget class="QTableView" name="tableViewQuery">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>600</width>
       <height>400</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>41</number>
     </attribute>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>80</y>
       <width>160</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit_mons">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>请输入月份</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_NeedActual">
        <property name="text">
         <string>NeedActual</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_NoCharge">
        <property name="text">
         <string>NoCharge</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
